packages/terra-alert/tests/nightwatch/AlertTitle.jsx

```diff
@@ -4,10 +4,9 @@ import Alert from '../../lib/Alert';
 
 const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
 
+// Alerts of the standard types with custom titles
 export default () =>
   <Base locale={locale}>
-    <h2>Alerts of the standard types with custom titles</h2>
```

Comment:
So in these examples these blurbs are to give context to the people viewing the link, i.e. for functional review. We will want to leave these in here. A possibility would be to add an additional div and then grab that div when taking screenshots instead of the default `[data-reactroot]` element.

Reply:
A lot of examples don't have instructional text. IMO these examples should be for automated tests and we should have other examples for manual validation if needed.

Reply:
Also, if we can validate in chrome that these never change, chrome should be the baseline to compare other browsers against for behavior...

Reply:
I lean towards keeping these examples for both automated tests and manual verification. I want to avoid duplicate examples as much as we can. The automated tests are only run in Chrome currently while our manual testing is done in all our supported browsers. I terms of instruction text, I'm not too concerned one way or the other if we have it or not. I know our solution designer has asked for it and found it helpful for verifying.

Reply:
Compromise, what if we log out instruction text in the console?

Reply:
I'm cool with that. :+1: